src/Components/CountdownTimer/CountdownTimer.js

```diff
@@ -15,17 +16,25 @@ import resetTimerIcon from '../../assets/timer/resetTimerIcon.png'
 const CountdownTimer = () => {
   const [ settings ] = useContext(SettingsContext)
   const [ view, setView ] = useContext(ViewContext)
+  const [ session, setSession ] = useContext(SessionContext)
   
+  const recordFocusInterval = (interval) => {
+    setSession({
+      ...session,
+      focusInterval: interval,
+      moodRating1: null,
+      contentSelected: null,
+      moodRating2: null,
+      restInterval: null,
+    })
+  }
+
   const timerDone = () => {
-    setView('content-selection')
+    setView('mood-rating-1')
```

Comment:
I really love this view context to control the application display and user flow.

Reply:
I really appreciate the Screen cast in the PR. That is SUPER helpful for understanding this code.